__tests__/integration-tests/routes/news.test.js

```diff
@@ -7,6 +7,17 @@ const regions = require('../../../src/data/regions.json');
 const app = createServer();
 const request = supertest(app);
 
+describe('News Api Route Sources', () => {
```

Comment:
This test block has already been added, Looks like I messed up the instructions :sweat_smile: 
Do the following:
- delete this describe block that you added
- add a describe block at the top, just after `const request = supertest(app)`
- add a description like `News API Route `
- and move all describe blocks inside that one

It should look like this:
```js
const app = createServer();
const request = supertest(app);

describe('News API Route', () => {
  describe('desc...', () => { tests... })
  describe('desc...', () => { tests... })
  describe('desc...', () => { tests... })
})
```

Reply:
Please have a look at latest commit